docs/content/latest/integrations/prisma.md

```diff
@@ -25,11 +27,11 @@ If YugabyteDB is installed, run the following `yb-ctl create` command to start a
 ./bin/yb-ctl create --tserver_flags=ysql_pg_conf="default_transaction_isolation=serializable"
 ```
 
-If you are new to YugabyteDB, you can be up and running with YugabyteDB in under five minutes by following the steps in [Quick start](/latest/quick-start/). After installing YugabyteDB, make sure to follow the step mentioned above.
+If you are new to YugabyteDB, you can be up and running with YugabyteDB in under five minutes by following the steps in [Quick start](../../quick-start/). After installing YugabyteDB, perform the preceding step.
```

Comment:
Does this make more sense to have at the top of the section, instead?

Reply:
Changed in DocMinorFixesForPR9105 branch